### PR TITLE
New date helper using momentjs 'fromNow' method

### PR DESCRIPTION
**Pull request by borisschapira**
Moment.js provides a quite useful function for relative time, it would be a shame not to use it ;)

### CODE REVIEW

**Comment by Xuanwo** [comment]
Looks wonderful~
Make it a question for @tommy351

File: lib/plugins/helper/date.js [hunk 1]
```diff
@@ -5,17 +5,21 @@ var isMoment = moment.isMoment;
 var isDate = require('util').isDate;
 
 function output(date, format, lang, timezone){
-  if (date == null) date = moment();
+  if (typeof myVar === 'undefined' || date === null) date = moment();
```
Comment (tommy351) [inline lib/plugins/helper/date.js]: `== null` can detect both `undefined` and `null`.

**Comment by tommy351** [comment]
It's not a good idea to write `fromNow` logic in the `output` function. You should separate it.

**Comment by borisschapira** [comment]
@tommy351 Is it better like this, only checking for `format` nullity in output (I dunno how to do it without changing something to `output`) ?

**Comment by tommy351** [comment]
I think you can rename the `output` function to `getDate` or `transformDate` and only let this function check types and apply locale and timezone and format the date in each helpers.

**Comment by borisschapira** [comment]
Better ?

**Comment by Xuanwo** [comment]
@tommy351

**Pull request merged**